sdk/src/lib/redeemer-proxy.ts

```diff
@@ -25,6 +26,8 @@ export default class OrangeKitTbtcRedeemerProxy implements TbtcRedeemerProxy {
 
   #sharesAmount: bigint
 
+  #builtDataStepCallback?: BuiltDataStepCallback
```

Comment:
Maybe we should rename to 🤔 :
```suggestion
  #buildDataStepCallback?: BuiltDataStepCallback
```

or

```suggestion
  #dataBuiltStepCallback?: BuiltDataStepCallback
```

Reply:
https://github.com/thesis/acre/pull/821/commits/8ceb6b4e73b454b77008971ce316228eaafffea2